packages/core/src/login/oidc/redirectIriUtils.ts

```diff
@@ -34,3 +34,16 @@ export function isValidRedirectUrl(redirectUrl: string): boolean {
     return false;
   }
 }
+
+export function removeOpenIdParams(redirectUrl: string): string {
+  const cleanedUpUrl = new URL(redirectUrl);
+  // For auth code flow
+  cleanedUpUrl.searchParams.delete("state");
+  cleanedUpUrl.searchParams.delete("code");
+  // For login error
+  cleanedUpUrl.searchParams.delete("error");
+  cleanedUpUrl.searchParams.delete("error_description");
+  // For RFC9207
+  cleanedUpUrl.searchParams.delete("iss");
+  return cleanedUpUrl.href;
```

Comment:
Is there a reason that the `id_token` and `access_token` are no longer being stripped? It appears that the test related to these parameters has been removed.

Reply:
Also, why not make this function just return a `URL` since you seem to just be wrapping most invocations of this function back in a `new URL`

Reply:
> Is there a reason that the id_token and access_token are no longer being stripped? It appears that the test related to these parameters has been removed.

These are only relevant for the implicit flow, which we don't support in the first place. Since we don't do the implicit flow authorization request, we don't need to handle the response from it.

> Also, why not make this function just return a URL since you seem to just be wrapping most invocations of this function back in a new URL

Good point, will do.